custom/apps/crs_reports/reports.py

```diff
@@ -189,9 +190,7 @@ def base_filters(self):
         return filters
 
     def date_to_json(self, date):
-        return tz_utils.adjust_datetime_to_timezone\
-            (date, pytz.utc.zone, self.timezone.zone).strftime\
-            ('%d-%m-%Y') if date else ""
+        return ServerTime(date).user_time(self.timezone).done().strftime('%d-%m-%Y') if date else ""
```

Comment:
Ah good catch, I was assuming the original code was correct when I was first going through, and didn't make the connection that this suffered from the same bug that was fixed and commented on in the other `date_to_json` function. Fixing!